docs/book/src/cronjob-tutorial/testdata/project/internal/controller/suite_test.go

```diff
@@ -92,25 +106,11 @@ var _ = BeforeSuite(func() {
 		Then, we start the envtest cluster.
 	*/
 
-	var err error
 	// cfg is defined in this file globally.
 	cfg, err = testEnv.Start()
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg).NotTo(BeNil())
 
-	/*
```

Comment:
Hi @mateusoliveira43 

I think we can reduce the scope of this one
We can have a PR to add the schemas before start the envtest

Then, we can have a PR that propose we remove apimachineryruntime and use  scheme.Scheme and explain why we want to do that

Reply:
done https://github.com/kubernetes-sigs/kubebuilder/pull/4466

will open next one after that one merges